### ISSUE TITLE
Feature request: "Typos: Send me pull request!" should point to the appropriate file

### ISSUE THREAD

**Issue by n9** [tpetricek/tomasp.net#12]
Currently, "Typos: Send me pull request!" points to the http://github.com/tpetricek/tomasp.net. 
It would be great if it would point to appropriate file. (It might be blocking for someone to correct small typos, since it might not be so easy to find the appropriate file.)

For instance, link in http://tomasp.net/blog/csharp-fsharp-async-intro.aspx/ would point to https://github.com/tpetricek/tomasp.net/blob/master/source/blog/csharp-fsharp-async-intro.aspx.html.

**Comment by tpetricek** [comment]
Yes, that would be great! I'd definitely accept a PR adding this - I was too lazy to implement it myself, because I thought it's minor issue, but I agree this would be useful.
